AlphaWallet/Tokens/Views/OpenSea/OpenSeaNonFungibleTokenViewCell.swift

```diff
@@ -67,6 +68,7 @@ class OpenSeaNonFungibleTokenView: UIView {
         imageHolder.clipsToBounds = true
 
         if let url = viewModel.imageUrl {
+            guard self.viewModel?.imageUrl != url else { return }
```

Comment:
Keeping in the `viewModel` around as a property isn't very clear (in the future). Do you think these 2 alternatives would be better:

A. instead of storing `viewModel`, store a `URL` instead with a name that includes it's currently loaded
B. store the URL in a backing property in `WebImageView` and perform a check in the `WebImageView.url` getter?